### ISSUE TITLE
the param community_tax's default value isn't correct of distr module

### ISSUE THREAD

**Issue by tianxiataiping** [okex/okchain#123]
## Summary of Bug
the param community_tax's default value is 0.00000000,it should be 0.02000000

## Steps to Reproduce

command input:

```
1、okchaincli query distr params -o json


```
result ：

```
{'community_tax': '0.00000000', 'withdraw_addr_enabled': True}

```

## Version

v0.10.1

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by RevelationOfTuring** [comment]
The community tax has a right display while replaying that on the commit  47a4ff62128b1aae73dee065527644dd19762702.

@tianxiataiping

**Comment by RevelationOfTuring** [comment]
It is fixed in v0.10.2 @tianxiataiping

**Comment by banishee** [comment]
It is better to use the latest code to build the `okchaind`. Then check if the value of the `community_tax` in the file `genesis.json` is `0.02`
